### DISCUSSION TITLE
What are all available IconElement

### DISCUSSION THREAD

**Discussion by AnesHamdani08** [HandyOrg/HandyControl#581] [Q&A] [answered]
I can set the style for a button to `Style="{StaticResource ButtonIcon}" Foreground="{DynamicResource PrimaryTextBrush}" hc:IconElement.Geometry="{StaticResource RightGeometry}"` which will show a right facing arrow , i found other `IconElement` such as , Calendar , Clock , UpDown , Left ... I tried to search the repository in GitHub which returned [This](https://github.com/HandyOrg/HandyControl/blob/master/doc/source/handycontrol/attach/iconElement/index.md) ,where can i find all the available `IconElement` from Handy Control

**Comment by ghost1372** [comment, marked as answer]
@AnesHamdani08  available geometries listed [here ](https://ghost1372.github.io/handycontrol/basic_xaml/geometries/)
also you can see a preview in demo app
![Untitled](https://user-images.githubusercontent.com/9213496/102013461-94100080-3d65-11eb-961b-31506e452755.png)

**Reply by AnesHamdani08** [reply]
Thanks
